addons/pvr.iptvsimple/pvr.iptvsimple.json

```
@@ -5,16 +5,15 @@
         {
             "type": "git",
             "url": "https://github.com/kodi-pvr/pvr.iptvsimple",
-            "branch": "Matrix",
-            "commit": "8381c866946a8a109e0c450aa0a029c3267f09c8"
+            "commit": "e9bdfa0aa7895e025dfcb0afee22d904f445b7be"
         }
     ],
     "build-options": {
         "no-debuginfo": true,
         "cflags": "-g0",
-        "cxxflags": "-g0"
+        "cxxflags": "-g0,-Wp,-U_GLIBCXX_ASSERTIONS"
```

```suggestion
        "cxxflags": "-g0 -Wp,-U_GLIBCXX_ASSERTIONS"
```

Maybe the correct syntax is like this. I am not sure.